### ISSUE TITLE
Issue with "ARIA: tab role": Formatting issue

### ISSUE THREAD

**Issue by enumiton** [mdn/content#11881]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role

#### What information was incorrect, unhelpful, or incomplete?
The formatting is missing a closing '`'. This causes the entire webpage to overflow while also making the paragraph unreadable.
> and `aria-selected="true" applied.

#### Specific section or headline?
Under the Example heading.

#### What did you expect to see?
and `aria-selected="true"` applied.

#### Did you test this? If so, how?
N/A

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/tab_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/tab_role/index.md
* Last commit: https://github.com/mdn/content/commit/0f1f6f74804725f379465c09f698b88611292c90
* Document last modified: 2021-12-01T06:01:14.000Z

</details>

**Comment by enumiton** [comment]
The link in the last paragraph under Precedence Order is also not formatted as a link and cannot be pressed.

> Screenreader support is still to be determined. Visit https://github.com/mdn/content/blob/main/rfcs/aria-roles.md for more information.

**Comment by enumiton** [comment]
Some of the inline-code is also missing double quotes which is not valid.
> `aria-selected=true`

Should be

> `aria-selected="true"`

**Comment by cw118** [comment]
I'd be happy to work on this issue if it hasn't been taken yet :)
